__tests__/components/viewers/__snapshots__/stop-viewer.js.snap

```diff
@@ -3199,56 +3209,46 @@ exports[`components > viewers > stop viewer should render countdown times for st
                       </IconWithText>
                     </button>
                   </div>
-                  <FromToLocationPicker
-                    label={true}
-                    onFromClick={[Function]}
-                    onToClick={[Function]}
+                  <span
+                    role="group"
```

Comment:
Why wrap this with a span? Could we add `role="group"` to `FromToLocationPicker` itself?

Reply:
If we did that then we'd either have to make all `FromToLocationPicker`s groups or add a new prop to turn it on and off which I'm not crazy about. This isn't perfect but I feel it's a happy middle ground